docs/overview/what-is-backstage.md

```diff
@@ -5,7 +5,7 @@ title: What is Backstage?
 description: Backstage is an open platform for building developer portals. Powered by a centralized software catalog, Backstage restores order to your microservices and infrastructure
 ---
 
-![software-catalog](https://backstage.io/blog/assets/6/header.png)
+<iframe width="672" height="378" src="https://www.youtube.com/embed/85TQEpNCaU0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Haha, accelerometer and gyroscope?! :) Like if I throw my phone into the wall in a fit of rage or so. Well OK.

Suggestion: could we keep the original image, and instead add this video between the "Backstage unifies" and "Out of the box" sections below?

Reply:
oh yeah, let's not remove that image 😄

Reply:
@freben accelerometer and gyroscope, definitely for VR/3D 😛  , but your suggestion could be a good metric for YouTube SEO 😂

Here we go, fixed it 

![image](https://user-images.githubusercontent.com/42934221/147096514-8c2cdb24-4b0b-4833-9263-21b3d044121c.png)